nvfuser/testing/__init__.py

```diff
@@ -0,0 +1,3 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
```

Comment:
I **think** we also need to include this in packaging. https://github.com/NVIDIA/Fuser/blob/main/setup.py ~ line 420 (I don't know the GUI isn't giving me a link)

Unfortunately, I don't think we have a test case that covers that in CI yet.

Reply:
I thought these are for C++ libraries. I don't see any entries for .py files. How do I confirm the packaging is OK?

Reply:
I think you are right. We have `packages=["nvfuser"]`, which should discover all directories with `__init.py__` already.

Testing this would need us to build a whl package and see if we have the testing utils shipped with that. I don't think we need to worry about it for now, sorry about the false alarm.